### ISSUE TITLE
508-defect-2 [HEADINGS]: "What you should know" on the sign up page should be a heading level 2

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22280]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user, I want to be able to navigate by headings to understand the structure of the page.

## Details

The "What you should know about signing up" string within the alert should be a `h2` as it is semantically more akin to a heading. This will help screen reader users better find and navigate to its contents on the page.

## Acceptance Criteria

- [ ] "What you should know..." is coded as a `h2`
- [ ] "What you should know..." is styled as a `h2`

## Steps to Recreate

1. Enter `https://staging.va.gov/health-care/covid-19-vaccine/stay-informed` in browser
2. Inspect the "What you should know..." string and confirm it's a `strong` not a `heading`

## Proposed Solution (if known)

See AC

## Screenshots or Trace Logs
![image](https://user-images.githubusercontent.com/14154792/112843452-95dda580-9070-11eb-8624-a5010592b295.png)
